src/pages/home/categories-list/category-item/category-item.js

```diff
@@ -12,6 +12,13 @@ const CategoryItem = ({
   language
 }) => {
   const [image, setImage] = useState(categoryImageUrl);
+  const history = useHistory();
+
+  const path = `/${categoryUrl}`;
+
+  const routeChange = () => {
```

Comment:
maybe rename this to 
```suggestion
  const changeRoute = () => {
```
it's sounds better to me.

Reply:
or
```suggestion
  const routeChangeHandler = () => {
```

Reply:
or
```suggestion
  const changeRouteHandler= () => {
```